### PR TITLE
small tweaks to candidate selection

### PR DESCRIPTION
**Pull request by coriolinus**

### CODE REVIEW

File: roadmap/implementors-guide/src/node/backing/candidate-selection.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ In the event that a parablock candidate proves invalid, this subsystem will rece
 
 ## Protocol
 
-Input: None
+Input: Parablock candidates and associated Proofs of Validity from collators
```
Comment (rphmeier) [inline roadmap/implementors-guide/src/node/backing/candidate-selection.md:15]: I've imagined that this subsystem will internally handle network logic for receiving these.

The other protocol sections refer to Input/Output in terms of messaging between subsystems, which this doesn't fall into the category of

Comment (coriolinus) [reply roadmap/implementors-guide/src/node/backing/candidate-selection.md]: OK, I've updated it.

**Comment by coriolinus** [comment]
CI is failing on the label check: it expects "B1-silent", not "B0-silent". I assert that this is a CI bug, not the fault of this PR.

**Comment by rphmeier** [comment]
cc @s3krit ^

**Pull request merged**